app/scripts/models/auth_brokers/base.js

```diff
@@ -263,6 +263,10 @@ define([
        */
       chooseWhatToSyncCheckbox: true,
       /**
+       * Should the broker convert links to text?
```

Comment:
`convert links` => `convert external links` for parity with the capability name.